api/src/main/java/io/opentelemetry/metrics/InstrumentWithBound.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.metrics;
+
+/**
+ * Base interface for all metrics with bounds defined in this package.
+ *
+ * @param <B> the specific type of Bound Instrument this instrument can provide.
+ * @since 0.1.0
+ */
+public interface InstrumentWithBound<B> extends Instrument {
```

Comment:
This should be called `InstrumentWithBinding`

Reply:
done.